pages/home/script.js

```diff
@@ -326,5 +362,32 @@ const onSearchInput = debounce(async ({ target }) => {
   suggestionOnClick();
   boxSuggestion.style.display = 'block';
   boxCategories.style.display = 'none';
-
 });
+
+// Load Hazard Detail Card
+let hazardReportID = '16cde280-ac58-467b-888e-dd0549274b6e';
+let currentReport = {};
+const body = document.getElementById('home-body');
+
+// Calculate distance from user to hazard with Haversine foruma
+function calcHazardDistance(lat1, lon1, lat2, lon2) {
```

Comment:
Can mostly be a helper function. Fine with it for now... I'll do the code polishing later if that's fine

Reply:
Ahh... good idea! sure we can look at that later.